app.py

```diff
@@ -74,17 +74,23 @@ async def join(ctx: Context):
     else:
         await ctx.send(f"{ctx.author} is not in a voice channel.")
 
+    return

```

Comment:
I need to test and see if these `return` statements prevent the "bot didn't send anything back" error messages.